javascript/src/ycell.ts

```diff
@@ -827,24 +827,32 @@ export class YCodeCell
   /**
    * Remove text from a stream output.
    */
-  removeStreamOutput(index: number, start: number): void {
-    this.transact(() => {
-      const output = this._youtputs.get(index);
-      const prevText = output.get('text') as Y.Text;
-      const length = prevText.length - start;
-      prevText.delete(start, length);
-    }, false);
+  removeStreamOutput(index: number, start: number, origin: any = null): void {
```

Comment:
What are valid types/values for `origin`?

Reply:
In Yjs an origin can be of [any type](https://github.com/yjs/yjs/blob/f0dc53f53f4c2fdf90a2701df73f73b6c2f06395/src/utils/Transaction.js#L51).